docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+# [.NET Core 2.2](#tab/netcore22)
+
+.NET Core 2.2 treats Linux as a single operating system. There's a single Linux build (per chip architecture) for supported Linux distributions.
+
+.NET Core 2.2 is supported on the following Linux distributions/versions:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                             |  Version                |  Architectures   |
+| ------------------------------ | ----------------------- | ---------------- |
+| Red Hat Enterprise Linux       |  6, 7                   | x64 |
+| CentOS                         |  7                      | x64 |
+| Oracle Linux                   |  7                      | x64 |
+| Fedora                         |  29, 30                 | x64 |
+| Debian                         |  9                      | x64, ARM32 |
+| Ubuntu                         |  16.04, 18.04, 18.10, 19.04    | x64, ARM32 |
+| Linux Mint                     |  17, 18                 | x64 |
+| openSUSE                       |  15+                    | x64 |
+| SUSE Enterprise Linux (SLES)   |  12 SP2+                | x64 |
+| Alpine Linux                   |  3.7+                   | x64 |
+
+For more information about .NET Core 2.2 supported OS, distributions, and lifecycle policy, see [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md).
+
+# [.NET Core 2.1](#tab/netcore21)
+
+.NET Core 2.1 treats Linux as a single operating system. There's a single Linux build (per chip architecture) for supported Linux distributions.
+
+.NET Core 2.1 is supported on the following Linux distributions/versions:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                             |  Version                |  Architectures   |
+| ------------------------------ | ----------------------- | ---------------- |
+| Red Hat Enterprise Linux       |  6, 7, 8                | x64 |
+| CentOS                         |  7+                     | x64 |
+| Oracle Linux                   |  7+                     | x64 |
+| Fedora                         |  29+                    | x64 |
+| Debian                         |  9                      | x64, ARM32 |
+| Ubuntu                         |  16.04, 18.04, 19.04, 19.10    | x64, ARM32 |
+| Linux Mint                     |  17+                    | x64 |
+| openSUSE                       |  15+                    | x64 |
+| SUSE Enterprise Linux (SLES)   |  12 SP2+                | x64 |
+| Alpine Linux                   |  3.7+                   | x64 |
+
+For more information about .NET Core 2.1 supported OS, distributions, and lifecycle policy, see [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md).
+
+---
+
+## Linux distribution dependencies
```

Comment:
How Is it decided which distros have their dependencies listed?  For example why aren't Debian and Alpine listed?

Reply:
This info was ported from the original docs. I'm working on getting the product team to own this dependency information so there is less people involved in keeping this up-to-date.

Reply:
FYI, I am pushing for https://github.com/dotnet/coreclr/issues/26303 to get fixed which I think would be helpful here.